java/ql/src/semmle/code/java/frameworks/play/Play.qll

```diff
@@ -0,0 +1,167 @@
+import java
+
+/**
+ * Play MVC Framework Result Class
+ */
+class PlayMVCResultClass extends Class {
```

Comment:
Abbreviations inside identifiers should use camelCase/PascalCase, so this should be `PlayMvcResultClass`.

Reply:
@aschackmull done renamed all to PascalCase.